test/quantization/test_quantize_fx.py

```diff
@@ -610,45 +613,92 @@ def forward(self, x):
         original_ref_m.conv2.weight = torch.nn.Parameter(original_m.standalone.conv.weight.detach())
         original_ref_m.conv2.bias = torch.nn.Parameter(original_m.standalone.conv.bias.detach())
 
-        qconfig_dict = {"": default_qconfig}
-        config_name = {"standalone_module_name": [("standalone", None, None)]}
-        config_class = {"standalone_module_class": [(StandaloneModule, None, None)]}
-        for prepare_config in [config_name, config_class]:
+        float_interface_config = {
+            "input_quantized_idxs": [],  # float input
+            "output_quantized_idxs": [],  # float output
+        }
+
+        quantized_interface_config = {
+            "input_quantized_idxs": [0],  # quantized input
+            "output_quantized_idxs": [0],  # quantized output
+        }
+        # is_name, is_float
+        options = itertools.product([True, False], [True, False])
```

Comment:
optional: 4 test cases instead of 1, to make it easier to fix things in the future (so the person fixing a bug doesn't have to comment out the test cases not relevant to the bug)

Reply:
sure

Reply:
will split to two cases since name and class are pretty similar